public/config/google.js

```diff
@@ -2,6 +2,7 @@ window.config = {
   routerBasename: '/',
   relativeWebWorkerScriptsPath: '',
   enableGoogleCloudAdapter: true,
+  supportsFuzzyMatching: true,
```

Comment:
This setting would need to be per `server` config?

Reply:
yeah, good point